lib/validators/specResolver.ts

```diff
@@ -832,7 +834,7 @@ export class SpecResolver {
    */
   private updateReferencesWithOneOf(
     subTreeMap: Map<string, PolymorphicTree>,
-    references: Array<{ readonly uri: unknown }>
+    references: StringMap<jsonRefs.UnresolvedRefDetails>
```

Comment:
why not use Map<string, UnresolvedRefDetails>

Reply:
because it's what we receive from `json-refs`. Not `Map`.